packages/nouns-webapp/src/pages/NotFound/index.tsx

```diff
@@ -1,13 +1,17 @@
-import { Col } from 'react-bootstrap';
+import { Col, Image } from 'react-bootstrap';
 import Section from '../../layout/Section';
 import classes from './NotFound.module.css';
+import _404img from '../../assets/404noun.png';
 
 const NotFoundPage = () => {
   return (
-    <Section bgColor="transparent" fullWidth={true}>
-      <Col lg={{ span: 8, offset: 2 }}>
+    <Section bgColor="transparent" fullWidth={false}>
+      <Col lg={4}>
+        <Image src={_404img} fluid />
+      </Col>
+      <Col lg={8}>
         <h1 className={classes.heading}>
-          404: This is not the person, place, or thing you're looking for ...
+          404: This is not the person, place, or thing you're looking for..
```

Comment:
nit: feels like we should have either `.` or `...` here. Have never seen `..`

Reply:
@cryptoseneca LGTM other than this! What do you think?

Reply:
three dots work too. lgtm!